templates/image-builder.yml

```diff
@@ -168,13 +168,46 @@ objects:
               value: "${DISTRIBUTIONS_DIR}"
             - name: QUOTA_FILE
               value: "/app/quotas/org_quotas.json"
+            - name: SYSLOG_SERVER
+              value: "localhost:5140"
+
+        - image: "fluentd-hec:1.2.13"
+          name: fluentd-sidecar
+          resources:
+            requests:
+              cpu: "${CPU_REQUEST}"
+              memory: "${MEMORY_REQUEST}"
+            limits:
+              cpu: "${CPU_REQUEST}"
+              memory: "${MEMORY_LIMIT}"
+          env:
+            - name: SPLUNK_HEC_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: splunk
+                  key: token
+                  optional: false
+            - name: SPLUNK_HEC_URL
+              valueFrom:
+                secretKeyRef:
+                  name: splunk
+                  key: url
+                  optional: false
```

Comment:
These havent been added to the insights namespaces yet, that needs to happen first.

Reply:
PR was just merged.